docs/releases/v0.11.0.rst

```diff
@@ -90,6 +90,11 @@
   -[new] It is now possible to read multi-file ``.tar.gz`` files using
     :func:`iread()`. [#2392]
 
+  -[new] Added parameter ``encoding`` which will force fread to recode the
+    input using the specified encoding before attempting to read it. The
+    recoding process uses standard python codecs, and is therefore
+    single-threaded.
```

Comment:
What about using words `decode`/`decoding` instead of `recode`/`recoding` here? Since we're only decoding the input and not encode it again, also, the progress bar message is `Decoding ...`.